yt_dlp/extractor/gamejolt.py

```diff
@@ -0,0 +1,540 @@
+            'id': str_or_none(song.get('id')),
+            'title': str_or_none(song.get('title')),
+            'url': str_or_none(song.get('url')),
+            'release_timestamp': int_or_none(song.get('posted_on'), scale=1000),
+        } for song in game_overview.get('songs') or []), game_id, traverse_obj(
+            game_overview, ('microdata', 'name'), (('twitter', 'fb'), 'title'), expected_type=str_or_none, get_all=False))
+
+
+class GameJoltCommunityIE(GameJoltPostListBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?gamejolt\.com/c/(?P<id>(?P<community>[\w-]+)(?:/(?P<channel>[\w-]+))?)(?:(?:\?|\#!?)(?:.*?[&;])??sort=(?P<sort>\w+))?'
+    _TESTS = [{
+        'url': 'https://gamejolt.com/c/fnf/videos',
+        'playlist_mincount': 50,
+        'info_dict': {
+            'id': 'fnf/videos',
+            'title': 'Friday Night Funkin\' - Videos',
+            'description': 'md5:6d8c06f27460f7d35c1554757ffe53c8'
+        },
+        'params': {
+            'playlistend': 50,
+            'ignore_no_formats_error': True,
+        },
+        'expected_warnings': ['skipping format', 'No video formats found', 'Requested format is not available'],
+    }, {
+        'url': 'https://gamejolt.com/c/youtubers',
+        'playlist_mincount': 50,
+        'info_dict': {
+            'id': 'youtubers/featured',
+            'title': 'Youtubers',
+            'description': 'md5:53e5582c93dcc467ab597bfca4db17d4'
+        },
+        'params': {
+            'playlistend': 50,
+            'ignore_no_formats_error': True,
+        },
+        'expected_warnings': ['skipping format', 'No video formats found', 'Requested format is not available'],
+    }]
+
+    def _real_extract(self, url):
+        display_id, community_id, channel_id, sort_by = self._match_valid_url(url).group('id', 'community', 'channel', 'sort')
+        if not channel_id:
+            channel_id = 'featured'
+        if not sort_by:
+            sort_by = 'new'
+
+        community_data = self._call_api(
+            'web/communities/view/%s' % community_id, display_id, note='Downloading community info', errnote='Unable to download community info')['community']
+        channel_data = traverse_obj(self._call_api(
+            'web/communities/view-channel/%s/%s' % (community_id, channel_id), display_id, note='Downloading channel info', errnote='Unable to download channel info', fatal=False), 'channel') or {}
+
+        title = '%s - %s' % (community_data.get('name'), channel_data['display_title']) if channel_data.get('display_title') else community_data.get('name')
+        description = self._parse_content_as_text(
+            self._parse_json(community_data.get('description_content', '{}'), display_id, fatal=False) or {})
+        return self.playlist_result(
+            self._entries('web/posts/fetch/community/%s?channels[]=%s&channels[]=%s' % (community_id, sort_by, channel_id), display_id, 'Downloading community posts', 'Unable to download community posts'),
```

Comment:
```suggestion
            self._entries(
                'web/posts/fetch/community/%s?channels[]=%s&channels[]=%s' % (community_id, sort_by, channel_id),
                display_id, 'Downloading community posts', 'Unable to download community posts'),
```
all these long lines can be cleaned up even more with fstrings